Makefile

```diff
@@ -1045,13 +1046,23 @@ ifeq ($(XZ), 0)
 endif
 
 .PHONY: bench-all
-bench-all:
+bench-all: bench-addons
 	@echo "Please use benchmark/run.js or benchmark/compare.js to run the benchmarks."
 
 .PHONY: bench
-bench:
+bench: bench-addons
 	@echo "Please use benchmark/run.js or benchmark/compare.js to run the benchmarks."
 
+# Build required addons for benchmark before running it.
+.PHONY: bench-addons
+bench-addons: benchmark/napi/function_call/build/Release/binding.node \
```

Comment:
Should this be `bench-addons-build` instead, similar to (for example) `lint-md-build`? `bench-addons` makes it sound like it runs addons benchmarks. This just builds them but does not run them.

Reply:
Good point.